### PR TITLE
update registry reference for circle

### PR DESCRIPTION
**Pull request by ABaldwinHunter**
This change should allow tests to pass on PRs from forks

And fix https://github.com/codeclimate/codeclimate-fixme/pull/8

@codeclimate/review

### CODE REVIEW

**Comment by mxie** [comment]
LGTM

File: circle.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ machine:
   environment:
     CLOUDSDK_CORE_DISABLE_PROMPTS: 1
     image_name: codeclimate-fixme
+    REGISTRY_ROOT: us.gcr.io/code_climate
 
 dependencies:
   pre:
```
Comment (GordonDiggs) [inline circle.yml]: The 4 lines in this section need to move to the master section below

**Comment by ABaldwinHunter** [comment]
@codeclimate/review updated following `codeclimate-rubocop` `circle.yml` - [gold standard](https://github.com/codeclimate/codeclimate-rubocop/blob/master/circle.yml) 

(I noticed that no tests actually get run atm, so we may want to add some in a different PR.)

**Comment by GordonDiggs** [comment]
LGTM

**Pull request merged**